packages/bs-ethereum/src/BSEthereum.ts

```diff
@@ -0,0 +1,180 @@
+import {
+  Account,
+  BSCalculableFee,
+  BSWithNameService,
+  BSWithNft,
+  BlockchainDataService,
+  BlockchainService,
+  ExchangeDataService,
+  Network,
+  NftDataService,
+  PartialBy,
+  Token,
+  TransferParam,
+} from '@cityofzion/blockchain-service'
+import { ethers } from 'ethers'
+import { DEFAULT_URL_BY_NETWORK_TYPE, NATIVE_ASSETS, TOKENS } from './constants'
+import { BitqueryEDSEthereum } from './BitqueryEDSEthereum'
+import { GhostMarketNDSEthereum } from './GhostMarketNDSEthereum'
+import { RpcBDSEthereum } from './RpcBDSEthereum'
+import { BitqueryBDSEthereum } from './BitqueryBDSEthereum'
+
+export class BSEthereum<BSCustomName extends string = string>
+  implements BlockchainService, BSWithNft, BSWithNameService, BSCalculableFee
+{
+  blockchainDataService!: BlockchainDataService
+  blockchainName: BSCustomName
+  feeToken: Token
+  exchangeDataService!: ExchangeDataService
+  tokens: Token[]
+  network!: Network
+  nftDataService!: NftDataService
+
+  private derivationPath = "m/44'/60'/0'/0/?"
```

Comment:
nitpick: Perhaps derivationPath for ETH should be exported from `'./constants'` 👀

Reply:
Maybe, I have just followed the same pattern of BSNeo3 and BSNeoLegacy, where we just have a private property. Should I also change in BSNeo3 and BSNeoLegacy ?